oca_dependencies.txt

```diff
@@ -0,0 +1,2 @@
+# See https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#oca_dependencies-txt
```

Comment:
FTR this file is not needed as dependencies are installed via PIP

Reply:
@isserver1 can you fix this ?